ls1intum/Artemis
# `Development`: Reduce the length of the commit_hash column in the submission table to 60

### Checklist
#### General


#### Changes affecting Programming Exercises

### Motivation and Context


While working on #4732 I noticed that the `commit_hash` column of the submission table was set to the type `varchar(255)`. As the commit hash is always 40 characters long this, therefore, wastes unnecessary space in the database.

### Description

After talking with @krusche we decided that this should be reduced to `varchar(60)`. The overhead of 20 characters is just a safeguard in case the commit hash is for some reason suddenly a bit longer. While should never happen, and I also checked that Bitbucket and Gitlab always return a commit hash of 40 chars, it's better to be safe than sorry.

### Steps for Testing

Prerequisites:
- 1 Instructor

1. Log in to Artemis
2. Check that the existing submissions are still working properly
3. Test that new submissions can be created

### Review Progress





#### Code Review
#### Manual Tests
